cmd/mesh/testdata/manifest-generate/output/pilot_override_kubernetes.yaml

```diff
@@ -18,6 +18,29 @@
 
 # NodeAgent component is disabled.
 
+apiVersion: v1
```

Comment:
why are there such huge diffs here? it seems like some resources are reordered.

Reply:
yep I think it doesn't makes sense, but as I said here: https://github.com/istio/operator/pull/482#pullrequestreview-308998258, seems like the output order is not stable